src/main/ipc.ts

```diff
@@ -15,10 +15,11 @@ import { join } from "path";
 import { debounce } from "shared/utils/debounce";
 
 import { IpcEvents } from "../shared/IpcEvents";
-import { setBadgeCount } from "./appBadge";
 import { autoStart } from "./autoStart";
 import { VENCORD_FILES_DIR, VENCORD_QUICKCSS_FILE, VENCORD_THEMES_DIR } from "./constants";
-import { mainWin } from "./mainWindow";
+import { globals } from "./mainWindow";
+// !!IMPORTANT!! ./appBadge import must occur after ./mainWindow
+import { setBadgeCount } from "./appBadge";
```

Comment:
consider inline requiring this module to fix the circular dependency issues in a cleaner way

Reply:
I'm surprised to hear casted `require` being called cleaner than just normal imports in a typescript context... but sure

Reply:
@Vendicated to confirm since we'd need the require hacks for settings as well, you'd prefer the first version over the second version below?

<img src="https://github.com/Vencord/Vesktop/assets/8976828/601b92d4-2bd0-4236-b9d1-7415183211b4"  width="50%" /><img src="https://github.com/Vencord/Vesktop/assets/8976828/baaaeaa9-5cf5-4217-b9a7-4b197a1ebad2"  width="50%" />

Reply:
If the current inclusion of `// eslint-disable-next-line simple-import-sort/imports` in `ipc.ts` is the roadblock on getting this merged, I'm happy to revert it to the `require` hacks. At this point I just want to not have to maintain my own branch on my system to keep up with updates...